File: Zerojudge/Python/b964.py
i = int(input())
n = list(map(int, input().split(' ')))
n.sort()
T = []
F = []
(Max, Min) = (max(n), min(n))
(worst, best) = (0, 0)

for t in range(0, i):
    if(n[t]>=60):
        T.append(n[t])

    else:
        F.append(n[t])

for t in range(0, i):
    print(n[t], end=' ')

if(len(T)!=0 and len(F)!=0):
    print('\n{}\n{}'.format(F[-1], T[0]))
    
else:
    print('\n{}\n{}'.format(F[-1], "worst case")) if(len(T)==0) else print('\n{}\n{}'.format("best case", T[0]))
    

File: Zerojudge/CPP/b964.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(0);

    int i;
    cin >> i;
    vector<int> n(i);
    for(int t = 0; t < i; t++) {
        cin >> n[t];
    }
    sort(n.begin(), n.end());
    vector<int> T, F;
    for(int t = 0; t < i; t++) {
        if(n[t] >= 60) {
            T.push_back(n[t]);
        } else {
            F.push_back(n[t]);
        }
    }
    for(int t = 0; t < i; t++) {
        cout << n[t] << ' ';
    }
    cout << '\n';
    if(!T.empty() && !F.empty()) {
        cout << F.back() << '\n' << T.front() << '\n';
    } else if(T.empty()) {
        cout << F.back() << '\n' << "worst case" << '\n';
    } else {
        cout << "best case" << '\n' << T.front() << '\n';
    }
    return 0;
}
